elide-contrib/elide-dynamic-config-helpers/pom.xml

```diff
@@ -100,16 +100,6 @@
             <artifactId>hjson</artifactId>
             <version>${hjson.version}</version>
         </dependency>
-        <dependency>
-            <groupId>org.junit.jupiter</groupId>
```

Comment:
Can you remove the version variables above as well?

Reply:
Removed.